slsa-provenance/main.go

```diff
@@ -153,6 +159,16 @@ func main() {
 		panic(err)
 	}
 
+	// Upload to tlog
+	rekorClient, err := rekor.NewClient(defaultRekorAddr)
+	if err != nil {
+		panic(err)
+	}
+	// TODO: Is it a bug that we need []byte(string(k.Cert)) or else we hit invalid PEM?
```

Comment:
weird. Does string() perform some sort of conversion from the cert bytes that eliminate some non-printable characters?

Reply:
I did try to figure this one out, and am going to create an issue if I get a little closer. It had me stuck for a very long time until I realized that tekton chains ends up sending this converted string successfully. I'll take a look, it might mean a bug upstream somewhere.